aradhakrishnanGFDL/CatalogBuilder
# Configuration yaml: To ease the config aspects and to help use catalog builder from within a python shell/script/notebook

First task-> https://github.com/aradhakrishnanGFDL/CatalogBuilder/issues/49

To test this-

we are assuming that intakebuilder package is installed and we loaded the environment.

test to make sure the old way of doing things still work
gen_intake_gfdl.py --overwrite /archive/am5/am5/am5f3b1r0/c96L65_am5f3b1r0_pdclim1850F/gfdl.ncrc5-deploy-prod-openmp/pp /nbhome/a1r/catalogbuilder/testoutput/testnew

gen_intake_gfdl.py --config /home/a1r/github/CatalogBuilder/configs/config.yaml

-use the scripts in the repository:

(uses --config)

scripts/gen_intake_gfdl_runner_config.py

this script runs it with the command line args, but you can run it from python shell or a notebook. simply run the script.
